docs/vectorizer_guide.md

```diff
@@ -62,11 +68,11 @@ SELECT ai.create_vectorizer(
 );
 ```
 
-In this example, the `contents` field is split into multiple chunks if it's
+In this example, if the `contents` field is lengthy, it is split into multiple chunks
```

Comment:
Lengthy is duplicated (it's on the next line, but I can't highlight that line in github's review).

Reply:
:-), thanks. Fixed.